buildsystem/scripts/EmbedWinDependencies.cmake

```diff
@@ -11,6 +11,21 @@ if(NOT dumpbin)
 	return()
 endif()
 
+# Paths for applocal and powershell
+set(APPLOCAL_SCRIPT "${vcpkg_dir}/../../scripts/buildsystems/msbuild/applocal.ps1")
+set(POWERSHELL_COMMAND "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe")
```

Comment:
I'm not certain about the hard-coded absolute path.
Can we use `powershell.exe` and let `execute_process` find the location?

Reply:
I'm not sure if it is in everyone's PATH variable, you can chose if you want to use old commandline or powershell as standard. Could be that it is not found, at least on my production system (not the vm) that is the case. :-S

Reply:
It's alright. This works for now. You can use `$ENV{SYSTEMROOT}/System32/WindowsPowerShell/v1.0/powershell.exe` to avoid the case of Windows installed in a different location.